app/experimenter/docs/adrs/0005-doc-hub.md

```diff
@@ -0,0 +1,151 @@
+# Use [TBD] for the Nimbus User Doc Hub
+
+- Status: proposed
+- Deciders: Lauren Zugai, Kate Hudson, [TBD]
+- Date: 2021-02-16
+
+Technical Story: [Documentation Hub for Nimbus Users](https://jira.mozilla.com/browse/EXP-860)
+
+## Context and Problem Statement
+
+In scaling Nimbus for 2021 and beyond, we need to provide experiment owners, data scientists, and desktop and Fenix engineers documentation to better streamline working with the platform. Following the same line of thought in [`nimbus-shared` ADR 0001](https://github.com/mozilla/nimbus-shared/blob/main/docs/adr/0001-one-console-app.md), users should expect a single source of truth for their documentation needs.
+
+## Decision Drivers
+
+- Docs should be kept in a central location
+- Finding relevant documentation should be easy, quick, and clear for all of our users, and cover their specific doc needs
+- Docs should be easily editable by anyone; maintenance should be straightforward and scalability should be taken into account
+- Hosting and deployment efforts should be very minimal
+
+## Considered Options
+
+- Mana / Confluence
+- Docusaurus + GH Pages
+- [Experimenter repo](https://github.com/mozilla/experimenter/) Wiki
+- Storybook
+
+## Decision Outcome
+
+Chosen option: "[TBD]", because [TBD].
+
+LZoog 2/16: I've gone back and forth on what I think would be best and at the moment, I'm leaning towards the Docusaurus + GH Pages option, but would like input from those that will actually be using these docs before making the call. While the "cons" can be mitigated by clear getting started docs, I'm not sure how heavily weighted some pros and cons are, like writing in MD, using GH, editing and committing doc files within the repository, or peer reviews of doc edits.
+
+### Positive Consequences
+
+- TBD, a couple sentences from the "Pros" section of the chosen option
+
+Note: we could technically point any of these options to a custom domain if we wanted, even if we just wanted to use a vanity URL to redirect to the documentation (for the Wiki option).
+
+### Negative Consequences
+
+- TBD, a couple sentences from the "Cons" section of the chosen option
```

Comment:
I don't think we need pros and cons of the chosen outcome considering they will be listed in the individual options.

Reply:
It's more like a one or two sentences summary of negative/positive consequences, but yeah point noted / won't be an exact copy and paste.